### DISCUSSION TITLE
Doxxed by GitHub Again

### DISCUSSION THREAD

**Discussion by scottgarner** [community/community#49644] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The "Confirm your account recovery settings" screen appeared while I was hosting a livestream, releasing my personal phone number to the viewers. This is not the first time this has happened and is tremendously frustrating.

**Comment by roncli** [comment]
This screen is incredibly frustrating, and makes using GitHub not an option for content creators.

What's worse is I do not want to see "Confirm your account recovery settings" in place of the URL I'm trying to visit, ever.  If I enter a URL, I expect the content that should be delivered by that URL.  If you want me to do something related to my account, make it a notification or a popup that will THEN bring up this screen WHEN I CLICK ON IT.

This has been this way long enough that I'm considering switching providers for my projects.  Please do something about this.
